### PR TITLE
Inventory payload request to use namespace

### PR DESCRIPTION
**Pull request by gildub**
The API call to the inventory-payload app has [changed](
https://github.com/konveyor/virt-payload/commit/785efd77c558961c4fbedc301a320eb9afd0dd34) slightly from:
`/api/v1/extract?providers=provider1,provider2`
to:
`/api/v1/extract?providers=namespace1/provider1,namespace2/provider2`

### CODE REVIEW

**Comment by gildub** [comment]
@mturley, @konveyor/virt-payload has been updated, could you please review?

**Comment by gildub** [comment]
Updated after http proxy addition.

**Review by mturley — CHANGES_REQUESTED** [review]
Sorry @gildub , I started a review and wrote this comment on Friday and then forgot to submit it. Once this change is made I'll approve and merge!

**Review by mturley — APPROVED** [review]

File: src/app/Providers/components/ProvidersTable/VMware/VMwareProvidersTable.tsx [hunk 1]
```diff
@@ -65,7 +65,9 @@ const VMwareProvidersTable: React.FunctionComponent<IVMwareProvidersTableProps>
 
   const inventoryDownloadURL = `${
     VIRT_META.inventoryPayloadApi
-  }/api/v1/extract?providers=${selectedItems.map((provider) => provider.name).join()}`;
+  }/api/v1/extract?providers=${selectedItems
+    .map((provider) => `${VIRT_META.namespace}/${provider.name}`)
```
[the remark this answers is not in the document]
Comment (gildub) [reply src/app/Providers/components/ProvidersTable/VMware/VMwareProvidersTable.tsx]: You're right. Thanks. Fixed.

**Pull request merged**